test/suite/test_durable_ts01.py

```diff
@@ -63,6 +63,7 @@ def skip(self):
             (self.ds.is_lsm() or self.uri == 'lsm')
 
     # Test durable timestamp.
+    @unittest.skip("Temporarily Disabled")
```

Comment:
Could you add a comment describing why the tests are disabled?

Reply:
Added a comment to describe why the recovery tests are failing. And these must be fixed with the following PR that replaces the data value with history store value.